pkg/cli/config/config.go

```diff
@@ -112,7 +140,11 @@ func (config BackplaneConfiguration) testHTTPRequestToBackplaneAPI() (bool, erro
 		http.DefaultTransport = &http.Transport{Proxy: http.ProxyURL(proxyURL)}
 	}
 
-	req, err := http.NewRequest("HEAD", config.URL, nil)
+	url, err := config.GetBackplaneURL()
+	if err != nil {
+		return false, err
+	}
```

Comment:
nit: Let's make this boolean evaluation concise

```suggestion
	if url, err := config.GetBackplaneURL(); err != nil {
		return false, err
	}
```